Sources/SwiftProtobuf/Message.swift

```diff
@@ -112,17 +155,33 @@ public protocol Message: CustomDebugStringConvertible {
   // They are implemented in the protocol, not in the generated structs,
   // so can be overridden in user code by defining custom extensions to
   // the generated struct.
+
+  /// Hash value generated from this message's contents, for conformance with
+  /// the `Hashable` protocol
   var hashValue: Int { get }
+
+  /// Textual representation of this message's contents suitable for debugging,
+  /// for conformance with the `CustomDebutStringConvertible` protocol.
```

Comment:
Actually it is:

> \[class name\]:
> \[serializeText()\]

Namely because if they are in a collection, it wouldn't be clear what they actual options were when dumping the collections description.